index.html

```diff
@@ -590,11 +663,18 @@ <h1>Terminology</h1>
         Thing Description, whereas a virtual entity is the
         composition of one or more Things.</dd>
       <dt>
-        <dfn>Thing Directory</dfn>
+        <dfn>Thing Description Directory</dfn>
```

Comment:
"TD Directory" is used above - please align.
Is it a "Thing Description" directory or a "Thing Directory"?